.github/renovate.json

```diff
@@ -2,6 +2,8 @@
   "extends": [
     "config:base",
     ":separateMultipleMajorReleases",
+    ":separatePatchReleases",
```

Comment:
Any special reason to activate this?

Reply:
This makes separate PRs for `1.2.3 -> 1.2.4` and `1.2.3 -> 1.3.0`.

It's not necessary with the proposed config. It would be necessary if you wanted to configure minor and patch updates separately (example below – in this case, we would automerge `0.1.2 -> 0.1.3` but require a human to merge `0.1.2 -> 0.2.0`).

```
    {
      "matchUpdateTypes": ["minor"],
      "matchCurrentVersion": "!/^[~^]?0/",
      "automerge": true,
      "automergeType": "branch"
    },
    {
      "matchUpdateTypes": ["patch"],
      "automerge": true,
      "automergeType": "branch"
    }
```

Reply:
Ok, thanks again! We probably can remove it then.